db/fixtures/notification_types.yml

```diff
@@ -174,3 +174,83 @@
   :expires_in: 24.hours
   :level: :error
   :audience: global
+- :name: vm_destroy_success
+  :message: 'Destroying Instance %{subject} completed successfully.'
```

Comment:
Not sure if the `instance` here is a good word if you name the notification as `vm_*`

Reply:
Hmm, ok, I can change that to `VM` or `Virtual Machine`.